be/src/exprs/string_functions.h

```diff
@@ -137,8 +141,37 @@ class StringFunctions {
     static void parse_url_close(
         doris_udf::FunctionContext*,
         doris_udf::FunctionContext::FunctionStateScope);
-};
 
+
+    static doris_udf::StringVal money_format(doris_udf::FunctionContext* context,
+                                                 const doris_udf::DoubleVal& v);
+
+    static doris_udf::StringVal money_format(doris_udf::FunctionContext* context,
+                                                 const doris_udf::DecimalVal& v);
+
+    static doris_udf::StringVal money_format(doris_udf::FunctionContext* context,
+                                                 const doris_udf::BigIntVal& v);
+
+    static doris_udf::StringVal money_format(doris_udf::FunctionContext* context,
+                                                 const doris_udf::LargeIntVal& v);
+
+    struct comma_moneypunct : std::moneypunct<char> {
```

Comment:
class name should be camel-case
```suggestion
    struct CommaMoneypunct : std::moneypunct<char> {
```

Reply:
OK